### DISCUSSION TITLE
LSP Autoformat

### DISCUSSION THREAD

**Discussion by jin-ahn** [LazyVim/LazyVim#204] [Q&A] [answered]
Is there a way to disable or enable the LSP Autoformat feature for only specific file types or regex file names?

**Comment by folke** [comment, marked as answer]
I just added this functionality in LazyVim.

See the docs on how to use this https://www.lazyvim.org/configuration/recipes#disable-autoformat-for-some-buffers

**Reply by jin-ahn** [reply]
@folke it looks like the `disable-autoformat-for-some-buffers` section is missing on the recipes page

**Reply by jin-ahn** [reply]
nvm. its under the "Tips" page

**Reply by folke** [reply]
right, yep I moved that :)

Recipes will be just specs.
Tips will be extended soon. That's why
